Resources/Prototypes/Maps/pebble.yml

```diff
@@ -40,6 +41,7 @@
             MedicalIntern: [ 2, 2 ]
             Paramedic: [ 1, 1 ]
             Psychologist: [ 1, 1 ]
+            MedicalBorg: [ 1, 1 ]
```

Comment:
I want to limit borgs to one type at roundstart and keep them themed with the station. Med borg was added specifically for Submarine because it was a medical focuses station.